src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/Example1.java

```diff
@@ -37,8 +38,8 @@ protected void m4() { }  // OK. No implementation.
    */
   public int m8() {return 2;}  // OK. Have javadoc on overridable method.
 
-  @Override
-  public String toString() {  // Violation. No javadoc for @Override method.
+  @Override // violation
```

Comment:
please vertically align comment `// violation` with `// OK. Have javadoc on overridable method.`

Reply:
Done

Reply:
Not done, I still see it as `@Override // violation` but should be something like `@Override            // violation`

Reply:
Sorry; Accidentally didn't add edit to commit.